src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import Heading from '../Heading'
+import NavBars from '../NavBar/NavBar'
+import ChartForm from '../ChartForm/ChartForm'
+import { StatisticsContainer } from './Statistics.style'
+
+const Statistics = (props) => {
+  return (
+    <>
+    <NavBars></NavBars>
```

Comment:
This can become self-closing tag like so `<NavBar />` 🙂

Reply:
Fixed